src/core/function_public.tf

```diff
@@ -96,7 +96,7 @@ module "function_public_staging_slot" {
   storage_account_access_key = module.function_public.storage_account.primary_access_key
 
   os_type                                  = "linux"
-  linux_fx_version                         = "NODE|14"
+  linux_fx_version                         = "NODE|18"
```

Comment:
This is right, we also have to prepare the PR to update the production slot above.